certification/engine/engine.go

```diff
@@ -41,6 +41,38 @@ type CheckRunner interface {
 	Results() runtime.Results
 }
 
+// Register all checks
```

Comment:
Something isn't right here... Why did this entire block of code change?

Reply:
I think it switched positions from below `NewForConfig(...)` to above it in the given file?

Reply:
yes, I moved the definition of these global vars to the top.

Reply:
Carry on. :)